### PR TITLE
Some clean code, using IntelliJ, in CucumberTestContext

### PR DESCRIPTION
**Pull request by pascalgrimaud**

### CODE REVIEW

File: src/main/resources/generator/server/springboot/cucumber/CucumberTestContext.java.mustache [hunk 1]
```diff
@@ -52,7 +50,7 @@ public final class CucumberTestContext {
   }
 
   public static List<String> getResponseHeader(String header) {
-    return Collections.unmodifiableList(lastQuery().responseHeaders().get(header));
+    return Collections.unmodifiableList(Objects.requireNonNull(lastQuery().responseHeaders().get(header)));
```
Comment (DamnClin) [inline src/main/resources/generator/server/springboot/cucumber/CucumberTestContext.java.mustache:53]: Not needed :)

File: src/main/resources/generator/server/springboot/cucumber/CucumberTestContext.java.mustache [hunk 2]
```diff
@@ -68,11 +66,10 @@ public final class CucumberTestContext {
         return 0;
       }
 
-      if (!(element instanceof JSONArray)) {
+      if (!(element instanceof JSONArray array)) {
```
Comment (DamnClin) [inline src/main/resources/generator/server/springboot/cucumber/CucumberTestContext.java.mustache:69]: Why introducing an unused variable?

Comment (pascalgrimaud) [reply src/main/resources/generator/server/springboot/cucumber/CucumberTestContext.java.mustache:69]: used few line after

File: src/main/resources/generator/server/springboot/cucumber/CucumberTestContext.java.mustache (hunk 1 above)
Comment (pascalgrimaud) [reply src/main/resources/generator/server/springboot/cucumber/CucumberTestContext.java.mustache:53]: I can revert this change, it was just intellij doesn't like it:

image

**Review by DamnClin — APPROVED** [review]

**Pull request merged**